CardPayments/src/main/java/com/paypal/android/cardpayments/VaultResult.kt

```diff
@@ -0,0 +1,16 @@
+package com.paypal.android.cardpayments
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+/**
+ * A result returned by [CardClient] when an a successful vault occurs.
+ *
+ * @param setupTokenId the id for the setup token that was recently updated
+ * @param status the status of the updated setup token
+ */
+@Parcelize
+data class VaultResult(
+    val setupTokenId: String,
```

Comment:
Nit - can we align with iOS on the naming for this property? I don't have a preference for `setupTokenId` vs just `setupToken` (which is on iOS). Whichever is more common in the PP docs I guess

Reply:
Yeah good call. It'll be more expressive if we use `setupTokenId` to call out that it's an identifier.

Reply:
This part in the [Swift Naming Guidelines](https://www.swift.org/documentation/api-design-guidelines/#naming) makes me feel like `setupTokenID` would work best on iOS.

![Screenshot 2023-08-10 at 9 59 35 AM](https://github.com/paypal/paypal-android/assets/58225613/93661267-2058-435e-ae83-a357abaee9bb)

Reply:
Awesome - I like that!

Reply:
Resolving, Victoria updated this on iOS 👏